docs/nodes/validate/add-a-validator.md

```diff
@@ -74,156 +75,332 @@ Once your validator’s start time has passed, you will see the rewards it may e
 
 That’s it!
 
-## Add a validator with API calls
+## Add a validator with AvalancheJS
+
+We can also add a node to the validator set using [AvalancheJS](https://docs.avax.network/apis/avalanchejs/). To add a node the Fuji Network, we’ll examine the workflow below.
+
+### Getting started
+
+To use AvalancheJS, you can clone the repo:
+```zsh
+$ git clone https://github.com/ava-labs/avalanchejs.git
+```
+
+or add it to an existing project:
+
+```zsh
+$ yarn add --dev avalanche
```

Comment:
why `--dev` here?

Reply:
I saw this workflow from a non-production point of view so it seems more accurate.

Open to changing

Reply:
https://classic.yarnpkg.com/lang/en/docs/cli/add/
`Using --dev or -D will install one or more packages in your [devDependencies](https://classic.yarnpkg.com/en/docs/dependency-types#toc-dev-dependencies).`

Are you suggesting developers to just use avalancheJS in devdependencies?